Mathlib/Geometry/Manifold/SmoothManifoldWithCorners.lean

```diff
@@ -1398,6 +1402,34 @@ theorem ContinuousWithinAt.extChartAt_symm_preimage_inter_range_eventuallyEq
   rw [← nhdsWithin_eq_iff_eventuallyEq]
   exact hc.nhdsWithin_extChartAt_symm_preimage_inter_range
 
```

Comment:
```suggestion

@[deprecated (since := "2024-11-26")] alias
extChartAt_target_union_comp_range_mem_nhds_of_mem := extChartAt_target_union_compl_range_mem_nhds_of_mem